example_pack_test.go

```diff
@@ -35,7 +35,8 @@ func ExamplePackManifest_imageV11() {
 	opts := oras.PackManifestOptions{
 		ManifestAnnotations: map[string]string{
 			// this timestamp will be automatically generated if not specified
-			// use a fixed value here in order to test the output
+			// use a fixed value here in order to test the output and make the
```

Comment:
nit: How about:
```suggestion
			// use a fixed value here to make the pack result reproducible
```
?